exercises/perfectShuffle.js

```diff
@@ -8,7 +8,15 @@
  */
 
 function perfectShuffle(array) {
-  // This is your job. :)
+  let firstHalf = array.splice(0,array.length/2);
```

Comment:
You'll want to use `array.slice` here because `array.splice` modifies `array` itself.